docs/reference/developer_documentation.md

```diff
@@ -21,12 +21,9 @@
 
 ## Developing
 
-To initialize your database, create your local configs file, enter your db uri, and run the migrations.
+To initialize your database, init $DB_URI (set in `.env`) and run the migrations.
 ```
-cp configs.py.dist configs.py
-
-echo "DB_URI = 'sqlite:////path/to/db.db'" > configs.py
-
+set -o allexport; source .env; set +o allexport
```

Comment:
I don't think this command works on Windows (cmd or PowerShell). Can or should we make it cross-platform?

Reply:
I think we have two options:
1) create an `env.bat` with the correct `set DB_URI="postgresql://..."` statements and update the documentation to direct the user differently based on what kind of system they have. This is a little more complicated because the Docker file *will* use the `.env` file so  this file will need to be kept up-to-date with `env.bat` if the user is on Windows.
2) use `dot-env` as @TShapinsky suggested to avoid creating an OS-dependent solution. This *should* be able to read as intended on a Windows system (though this would need testing). We'd update the docs to avoid explicitly "source"-ing the environment file and just let the `dot-env` package handle it. I like this option better but I don't have ready access to a Windows system to test it. Is this something you could take a quick look at, @MatthewSteen?